ts/features/bonus/cgn/screens/CgnDetailScreen.tsx

```diff
@@ -77,7 +81,7 @@ const CgnDetailScreen = (props: Props): React.ReactElement => {
   const canDisplayEycaDetails =
     canEycaCardBeShown(props.eycaDetails) && props.isCgnEnabled;
 
-  return props.cgnDetails || props.isCgnInfoLoading ? (
+  return (
```

Comment:
🐞 
https://user-images.githubusercontent.com/822471/149983914-5d5ce813-b8c1-43e3-8d3e-556833b00051.mov


how to reproduce
- run dev-server
- run app
- activate CGN
- back to the wallet
- turn off dev server (simulating no backend response)
- loading/error
- user has to kill the app

Reply:
this was really a awesome catch, eae1bab to solve

Actually we came from a `pot.Some` than updated to `pot.SomeError`. The infinite load was caused by the Card Webview background that never updates to stop loading since it is mounted and unmounted immediately